Makefile

```diff
@@ -137,6 +137,12 @@ protobuild:
 	@protoc-go-inject-tag -input=./internal/credential/store/credential.pb.go
 	@protoc-go-inject-tag -input=./internal/credential/vault/store/vault.pb.go
 
+
```

Comment:
Is there a reason you want all this white space around these? If they are being isolated perhaps a comment as on line 146...

Reply:
fair enough.  adding comment.